assets/js/components/TourTooltips.test.js

```diff
@@ -369,5 +370,52 @@ describe( 'TourTooltips', () => {
 				1
 			);
 		} );
+
+		it( 'accepts a function to generate the event category', async () => {
+			const gaEventCategory = ( viewContext ) => `${ viewContext }_test`;
+			const expectedCategory = gaEventCategory( '' );
```

Comment:
Instead of testing it with an empty context, let's better add the context provider and test that the component can correctly read the context value.

Reply:
Good idea